src/main/scala/mesosphere/marathon/core/task/tracker/impl/InstanceStateOpProcessorDelegate.scala

```diff
@@ -1,64 +0,0 @@
-package mesosphere.marathon
-package core.task.tracker.impl
-//scalastyle:off
-import java.time.Clock
-
-import akka.Done
-import akka.actor.ActorRef
-import akka.util.Timeout
-import mesosphere.marathon.core.instance.Instance
-import mesosphere.marathon.core.instance.update.{ InstanceUpdateEffect, InstanceUpdateOperation }
-import mesosphere.marathon.core.task.tracker.impl.InstanceTrackerActor.ForwardTaskOp
-import mesosphere.marathon.core.task.tracker.{ InstanceStateOpProcessor, InstanceTrackerConfig }
-import mesosphere.marathon.state.Timestamp
-import org.apache.mesos
-
-import scala.concurrent.Future
-import scala.concurrent.duration._
-import scala.util.control.NonFatal
-//scalastyle:on
-/**
-  * Implements the [[InstanceStateOpProcessor]] trait by sending messages to the TaskTracker actor.
-  */
-private[tracker] class InstanceStateOpProcessorDelegate(
-    clock: Clock,
-    conf: InstanceTrackerConfig,
-    instanceTrackerRef: ActorRef)
-  extends InstanceStateOpProcessor {
-
-  import scala.concurrent.ExecutionContext.Implicits.global
-
-  private[impl] implicit val timeout: Timeout = conf.internalTaskUpdateRequestTimeout().milliseconds
```

Comment:
I guess we can remove this parameter from the config and use `instanceTrackerQueryTimeout` for all requests within the InstanceTracker?

Reply:
Oh. I thought it's the same. Yeah, I'm using `instanceTrackerQueryTimeout` for all. Good catch. If I remove it we technically break the Marathon process options :/.

Reply:
we break on non-existing process options?

Reply:
> Hm. Technically this config param was hidden and marked for internal use.

We have an option string for internal use?